<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="drawableArea">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>551</width>
      <height>271</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
   </widget>
   <widget class="chosenColor" name="firstColorBox" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">:hover
{
	background-color: rgba(20,40,50,0.5);
}
</string>
    </property>
   </widget>
   <widget class="chosenColor" name="secondColorBox" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">:hover
{
	background-color: rgba(20,40,50,0.5);
}

</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="menu">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFiles"/>
   <addaction name="separator"/>
   <addaction name="actionTools"/>
   <addaction name="separator"/>
   <addaction name="actionView"/>
  </widget>
  <widget class="QToolBar" name="barTools">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="barView">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionBigger"/>
   <addaction name="actionSmaller"/>
  </widget>
  <action name="actionBigger">
   <property name="text">
    <string>Bigger</string>
   </property>
   <property name="toolTip">
    <string>Bigger</string>
   </property>
  </action>
  <action name="actionTools">
   <property name="text">
    <string>Tools</string>
   </property>
   <property name="toolTip">
    <string>Tools</string>
   </property>
  </action>
  <action name="actionFiles">
   <property name="text">
    <string>Files</string>
   </property>
   <property name="toolTip">
    <string>Files</string>
   </property>
  </action>
  <action name="actionView">
   <property name="text">
    <string>View</string>
   </property>
   <property name="toolTip">
    <string>View</string>
   </property>
  </action>
  <action name="actionSmaller">
   <property name="text">
    <string>Smaller</string>
   </property>
   <property name="toolTip">
    <string>Smaller</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>chosenColor</class>
   <extends>QWidget</extends>
   <header location="global">chosencolor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
